showcase/src/app/demo/pretty-print/demo-pretty-print.component.html

```diff
@@ -0,0 +1,58 @@
+<example-viewer [extensions]="[]"
+				filesPath=""
+				exampleTitle="SHOWCASE.DEMO.PRETTY-PRINT.TRY_IT_YOURSELF">
+	<form>
+		<div class="pretty-print-input mb2">
+			<mat-form-field>
+				<textarea name="unformattedData"
+					   	  [(ngModel)]="unformattedData"
+						  matInput
+						  rows="17"
+					   	  placeholder="{{ 'SHOWCASE.DEMO.PRETTY-PRINT.ENTER_DATA' | translate }}">
```

Comment:
You can achieve the same with a shorter syntax 😉 

`[placeholder]="'SHOWCASE.DEMO.PRETTY-PRINT.ENTER_DATA' | translate"`